.eslintrc.json

```diff
@@ -1,7 +1,11 @@
 {
   "env": {
     "browser": true,
-    "es2021": true
+    "es2021": true,
+    "jest": true
+  },
+  "globals": {
+    "page": true
```

Comment:
I think there ought to be a way to only allow this `page` global in `*.test.js` files but I'm not sure what it is